x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/documents/promoted_documents.test.tsx

```diff
@@ -89,7 +136,7 @@ describe('PromotedDocuments', () => {
       expect(actions.clearPromotedIds).toHaveBeenCalled();
     });
 
-    describe('draggging', () => {
+    describe('dragging', () => {
```

Comment:
🤠

Reply:
Please pardon my spelling errors, I appreciate you flagging them.

Reply:
😇